### DISCUSSION TITLE
Updating packages' version in a pluto notebook

### DISCUSSION THREAD

**Discussion by gianmariomanca** [fonsp/Pluto.jl#1404] [Questions about using Pluto] [answered]
I was initially skeptical of Pluto's package management (slowed down things for me) until I had to share a complex pluto notebook with a colleague the afternoon before leaving for a trip. With a quick call he was able to install Julia and run the notebook with no issues: in ~30 mins we were deep diving into the details of the actual analysis.
So I'm on board now :) But what if I want to test the notebook with updated packages from time to time, and then if it works have the notebook remember the new updated versions, so I can commit it to a repository and share it again?
What are the appropriate/best steps?

**Comment by Pangoraw** [comment, marked as answer]
From the wiki: https://github.com/fonsp/Pluto.jl/wiki/%F0%9F%8E%81-Package-management#updating-packages
When updating, it will create a backup in case the new versions don't work as expected.
